sys/src/9/amd64/archamd64.c

```diff
@@ -76,7 +76,8 @@ cpuidhz(uint32_t *info0, uint32_t *info1)
 
 	//print("CPUID Vendor: %s\n", (char *)&info0[1]);
 	//print("CPUID Signature: %d\n", info1[0]);
-	if(memcmp(&info0[1], "GenuntelineI", 12) == 0){
+	if((memcmp(&info0[1], "GenuntelineI", 12) == 0) ||
+		(memcmp(&info0[1], "GenuineIntel", 12) == 0)) {
```

Comment:
This is not really right. You need to go back and see how the cpuid is turning ebx, ecx, and edx into info. The plan 9 usage mean that the info[] array was not a good fit to the string, and we ended up with this weird ordering. 

IOW, you should only need one of these strings, not both. This code is wrong.